conf/analysis/trip_model.conf.json.sample

```diff
@@ -8,6 +8,56 @@
             "similarity_threshold_meters": 500,
             "apply_cutoff": false,
             "incremental_evaluation": false
+        },
+        "gbdt": {
+            "feature_list": {
+                "data.user_input.mode_confirm": [
+                    "drive",
+                    "bike",
+                    "walk",
+                    "transit"
+                ],
```

Comment:
As we discussed, we should not be using `mode_confirm` and `purpose_confirm` to infer the replaced mode because in practice people either label all or none. So we are unlikely to have the `mode_confirm` and `purpose_confirm` for trips that we want to predict the replaced mode for.